detectors/aws/ec2/version.go

```diff
@@ -21,6 +21,9 @@ func Version() string {
 }
 
 // SemVersion is the semantic version to be supplied to tracer/meter creation.
+//
+// Deprecated: SemVersion() does not return the version with the `sem:` prefix.
+// Please use Version() instead.
```

Comment:
Please apply this suggestion to all places:

```suggestion
// Deprecated: Use [Version] instead.
```

Reply:
Just done. thanks for your supports @pellared